receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_metadata.go

```diff
@@ -21,6 +21,7 @@ import (
 // network metric constants
 
 const (
+	deviceLabelName    = "device"
```

Comment:
Maybe this label could be called `interface` instead? That way it's consistent with the config option.